components/mpas-albany-landice/src/mode_forward/Makefile

```diff
@@ -4,9 +4,11 @@
 OBJS =  mpas_li_core.o \
     mpas_li_core_interface.o \
     mpas_li_time_integration.o \
-    mpas_li_time_integration_fe.o \
```

Comment:
Because this module has been renamed, do we need to update the cmake build system that E3SM uses?  @jonbob , do you know how to do that?

Reply:
I believe we just need to modify ~/components/mpas-albany-landice/src/landice.cmake